temboardagent/postgres.py

```diff
@@ -53,6 +53,7 @@ def query_scalar(self, query, vars=None):
 class ConnectionManager(object):
     def __init__(self, postgres):
         self.postgres = postgres
+        self.app = self.postgres.app
```

Comment:
I'd suggest to add an `app` argument to `__init__` instead of getting it from `postgres` argument. That would more explicit IMHO.